rules/constants.go

```diff
@@ -0,0 +1,78 @@
+//Package rules ...
+// Copyright (c) 2017 Intel Corporation.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package rules
+
+// constants for rule file paths
+const (
```

Comment:
You can't have constants for file paths in file that can be imported by user, can you please move it to example

Reply:
these constants are specific to rules pkg only. can be moved to rules.go file instead of example.

Reply:
ok removed the constants.go Instead of separated example dir i have moved them to appropriate files and path can be configured in ngicdp.go